### DISCUSSION TITLE
Github bug:  Says there is a merge conflict when there is none.

### DISCUSSION THREAD

**Discussion by tzjames** [community/community#77945] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have a PR that is on a branch that is 1 ahead of master (0 behind).  
<img width="383" alt="Screenshot 2023-12-05 at 3 14 23 PM" src="https://github.com/community/community/assets/950231/d3f22602-cebd-46ae-9153-5c9871801edf">
However github says there is a merge conflict.
<img width="438" alt="Screenshot 2023-12-05 at 3 14 45 PM" src="https://github.com/community/community/assets/950231/6ac565fb-0cdb-4b0d-bf00-3bf543e5de15">

**Comment by jagthedrummer** [comment]
Here's a link to a public repo where this is happening. It's a tiny little change, and the branch for the PR was cut directly from `main`. The false positive on the merge conflict is also preventing any GitHub Actions workflows from running.

https://github.com/bullet-train-co/bullet_train-core/pull/713

**Reply by jagthedrummer** [reply]
I tried closing the PR and reopening it and that got GitHub to realize that there's actually not a conflict. 🤷
